rx-dom/index.d.ts

```diff
@@ -137,6 +137,6 @@ declare module Rx.DOM {
     }
 }
 
-declare module "rx.DOM" {
+declare module "rx-dom" {
```

Comment:
i have no idea. could not find any documentation for the ones with the `.` in the name.